docs/admin/setup/pyris.rst

```diff
@@ -31,70 +25,214 @@ E.g.: ``git clone https://github.com/ls1intum/Pyris.git Pyris``
-E.g.: ``cp Pyris/application.example.yml application.yml``
+Create an ``application.local.yml`` file using the provided ``application.example.yml`` in the Pyris repository as a base.
+E.g.: ``cp Pyris/application.example.yml application.local.yml``
 
-Now you need to configure the ``application.yml`` file. Here is an example configuration:
+Now you need to configure the ``application.local.yml`` file. Here is an example configuration:
 
 .. code-block:: yaml
 
-    pyris:
-        api_keys:
-          - comment: Artemis
-            llm_access: [AZURE_GPT35_TURBO_16K_0613, OPENAI_GPT35_TURBO_16K_0613]
-            token: super-secret
-        cache:
-            hazelcast:
-                host: cache
-                port: 5701
-        llms:
-            AZURE_GPT35_TURBO_16K_0613:
-                description: GPT-3.5 on Azure model from 2023-06-13 with 16k context length
-                name: GPT-3.5 (16k, Azure)
-                llm_credentials:
-                    api_type: azure
-                    api_base: 'https://<your-url>.azure.com/'
-                    api_version: 2023-03-15-preview
-                    deployment_id: gpt-35-16k
-                    model: gpt-3.5-turbo
-                    token: <token>
-            OPENAI_GPT35_TURBO_16K_0613:
-                description: GPT-3.5 on OpenAI from 2023-06-13 with 16k context length
-                name: GPT-3.5 Turbo (16k, OpenAI)
-                llm_credentials:
-                    chat_mode: 'True'
-                    model: gpt-3.5-turbo-16k-0613
-                    token: <token>
-
-Create Environment Variables File
+    api_keys:
+      - token: "secret"
+
+    weaviate:
+      host: "localhost"
+      port: "8001"
+      grpc_port: "50051"
+
+    env_vars:
+        test: "test"
+
+
+Create LLM Config File
 """""""""""""""""""""""""""""""""
+Create an ``llm_config.local.yml`` file using the provided ``llm_config.example.yml`` in the Pyris repository as a base.
+E.g.: ``cp Pyris/llm_config.example.yml llm_config.local.yml``
 
-Create a ``.env`` file containing the following variables:
+This file contains the configuration for the Large Language Model (LLM) that can be used by the pipelines in Pyris.
+
+The capability system in Pyris will use this configuration to determine which model to use for a pipeline.
+
+Now you need to configure the ``llm_config.local.yml`` file.
+
+Example OpenAI
+**************
+.. code-block:: yaml
+
+    - api_key: <your_openai_api_key>
+      capabilities:
+        context_length: 16385
+        gpt_version_equivalent: 3.5
+        image_recognition: false
+        input_cost: 0.5
+        json_mode: true
+        output_cost: 1.5
+        privacy_compliance: false
+        self_hosted: false
+        vendor: OpenAI
+      description: GPT 3.5 16k
+      id: oai-gpt-35-turbo
+      model: gpt-3.5-turbo
+      name: GPT 3.5 Turbo
+      type: openai_chat
```

Comment:
Please explain the properties in detail.

Reply:
Added a new section where we explain the every parameter